mappings/net/minecraft/block/IBlockInfoProxy.mapping

```diff
@@ -57,8 +57,13 @@ CLASS none/asl net/minecraft/block/IBlockInfoProxy
 		ARG 1 pos
 	METHOD e allowsLightTransmission ()Z
 	METHOD g getMapColor ()Lnone/ays;
-	METHOD h ()Z
+	METHOD h isFullBoundsCube2 ()Z
 	METHOD i getRenderType ()Lnone/api;
+	METHOD j ambientOcclusionLightLevel ()F
+	METHOD k isFullBoundsCubeForRendering ()Z
+	METHOD l isBasicBlock ()Z
```

Comment:
A "basic block" is a full cube, opaque, with no special redstone interaction.

It seems to be used by things that want to interact with a block that's not special. 
Bats use it for checking where they can roost, comparators use it to see if redstone passes through, tripwire checks to see if it can attach to it, ladders check that they can attach to it, etc.